platform/app/prisma/migrations/20260804120002_idempotency_receipt/migration.sql

```
@@ -0,0 +1,49 @@
+    "responseBody" JSON,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "expiresAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "IdempotencyReceipt_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "IdempotencyReceipt_scopeId_key_key" ON "IdempotencyReceipt"("scopeId", "key");
+
+-- CreateIndex
+CREATE INDEX "IdempotencyReceipt_expiresAt_idx" ON "IdempotencyReceipt"("expiresAt");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)